### DISCUSSION TITLE
Screen editor?

### DISCUSSION THREAD

**Discussion by nagyrobi** [HASwitchPlate/openHASP#77] [Q&A]
I see the project is based on [LVGL](https://github.com/lvgl/lvgl) but I don't see any compatible screen editor we could use. A quick glance shows that the existing LVGL editors output screens in C language not jsonl...

**Comment by fvanroie** [comment]
LVGL just released a [beta v0.1 of their new EdgeLine editor](https://forum.lvgl.io/t/v0-1-beta-is-out/4818/4) for lvgl. Those project file are in json format.

**Reply by nagyrobi** [reply]
Thanks... well it looks like a windows-only tool which requires cloud access... with a max 3-pages limit on the beta. I assume this plans to be a closed source something you need to pay for in the future...?

**Reply by fvanroie** [reply]
It is cross-platform I believe. I don't know about the license, you can ask in the [lvgl forum](https://forum.lvgl.io/t/introducing-lvgls-ui-editor-edgeline/4521/32).
This is the official lvgl editor, maintained by them... It's the closest thing to a usable editor I've seen so far.
